.github/workflows/continuous-integration.yml

```diff
@@ -42,12 +42,12 @@ jobs:
           ORG_GRADLE_PROJECT_epApiVersion: ${{ matrix.epVersion }}
         uses: eskatos/gradle-command-action@v1
         with:
-          arguments: verGJF build
+          arguments: verGJF build -x :jmh:test
         if: matrix.java == '8'
       - name: Build and test using Gradle and Java 11
         uses: eskatos/gradle-command-action@v1
         with:
-          arguments: :nullaway:test
+          arguments: :nullaway:test :jmh:test
```

Comment:
Just to clarify, does this run the performance test with caffeine? Or just the single sample file test? (I would have expected the caffeine run to be slower than what I am seeing for the CI jobs for this PR)

Reply:
It only runs the single unit test `NullawayJavacTest`.  Running the performance test with Caffeine requires running `./gradlew jmh`, and the standard `build` task does not depend on the `jmh` task.